docs/bounty-development-guide.md

```diff
@@ -0,0 +1,47 @@
+# Bounty Development Guide
+
+Hi, Awesome people! Welcome to start a bounty task on Mask Network.
+
+## Tech Reqirements
+
+Mask Network extension is mainly written in TypeScript. The UI part is building with React upon [MUI](https://mui.com/getting-started/installation/) framework. Also, we write [CSS-in-JS](css-in-js.md). Use webpack to build the project. Besides that, we prefer widely adopting tech solutions that includes:
+
+- [Web3.js](https://web3js.readthedocs.io/) Ethereum JavaScript API
+- [react-use](https://streamich.github.io/react-use/) React Hooks — 👍
+- [bignumber.js](https://mikemcl.github.io/bignumber.js/) A JavaScript library for arbitrary-precision arithmetic.
+- [lodash](https://lodash.com/docs/) A modern JavaScript utility library delivering modularity, performance & extras.
+- [urlcat](https://urlcat.dev/) A URL builder library for JavaScript.
+
+> If your bounty task is related to another project, it could have extra requirements that need to consider.
+
+The codebase is totally open sourced. Anyone can review it. If you are familiar with these libraries mentioned above, it will take less friction for you to get start.
```

Comment:
```suggestion
If you are familiar with these libraries mentioned above, it will take less effort for you to get started.
The codebase is open-sourced under the AGPLv3 license.
```